src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -749,6 +843,13 @@ components:
         - dispatched
         - not-delivered
 
+    AedSearchBy:
+      title: AedSearchBy
+      description: An enumeration of search by fields for searching AEDs
+      type: string
+      enum:
+        - surname
```

Comment:
I've not checked, but is this the same enum value name/terminology that we use in VAC?
I know they are different search APIs, but we need to be consistent. If it is `surname` in both APIs, then happy days 😁 
But if VCA already uses some other term such as `lastName` then we should use it here too. We just need to check we are being consistent

Reply:
yes, from vca,

<img width="443" alt="image" src="https://user-images.githubusercontent.com/73763010/233606003-23472c25-ef8d-46c9-a03d-83ef259e9be5.png">

Reply:
Perfect ; thanks for checking 👍